### ISSUE TITLE
Booking payment on multiple realm & factions

### ISSUE THREAD

**Issue by iElsha** [Icecrown-Community/ICC-Member-Website-Public#6]

**Comment by githubtest1234567** [comment]
maybe we should add the option for balance payment too

**Comment by Bgabor997** [comment]
maybe we should add an option for balance payment too

**Comment by iElsha** [comment]
> maybe we should add an option for balance payment too

Yes, that would improve https://github.com/Icecrown-Community/ICC-Member-Website-Public/issues/25

**Comment by Dejnaa** [comment]
So, since you cancelled my post that had solutions and examples - I will ask here
Did the image look like something that could be made?
I explained to Choppa yesterday on voice and he did get the idea of thous changes

**Comment by iElsha** [comment]
Sorry, I forgot to move the message here.

![image](https://user-images.githubusercontent.com/38140638/105606126-c155c280-5d97-11eb-93f7-ba281b943229.png)

It's what I planned but not only for GC. For everyone.

**Comment by Dejnaa** [comment]
This has been added for M+ and for PvP - can we copy it to the Raid booking screen? :)

**Comment by Dejnaa** [comment]
@Bgabor997 Can we?

**Comment by Bgabor997** [comment]
we can, but this is bigger work too, i would say 5-6 hour bcs i need to change the db structure, and the submit method for this, and need to extend tests, both front and backend

**Comment by Dejnaa** [comment]
@Papa2324 Do you feel like this is something that should be bumped up on the priolist?

 (I have no access to that so I cant make any type of assessment)

**Comment by Papa2324** [comment]
nah, theres many other things id like to have before this one. unless it takes no time to do then sure
